src/koheesio/categorical_encoding.py

```diff
@@ -0,0 +1,30 @@
+# Import necessary libraries
+from sklearn.preprocessing import OneHotEncoder
```

Comment:
You're introducing a new dependency here, this should be included in the pyproject.toml.

Reply:
I have added dependency in the pyproject.toml